mono/metadata/image.c

```diff
@@ -1723,21 +1722,41 @@ find_by_guid (gpointer key, gpointer val, gpointer user_data)
 		data->res = image;
 }
 
+static MonoImage *
+mono_image_loaded_by_guid_internal (const char *guid, gboolean refonly);
+
 /**
  * mono_image_loaded_by_guid_full:
  */
 MonoImage *
 mono_image_loaded_by_guid_full (const char *guid, gboolean refonly)
 {
+	return mono_image_loaded_by_guid_internal (guid, refonly);
+}
+
+/**
+ * mono_image_loaded_by_guid_internal:
+ *
+ * Do not use.  Looks only in the global loaded images hash, will miss Assembly
```

Comment:
Shouldn't this comment be somewhere that it'll show up in the documentation? The same caveat applies to anyone using the external version.